src/VisualStudio/IntegrationTest/IntegrationTestBuild.slnf

```diff
@@ -0,0 +1,97 @@
+{
+  "solution": {
+    "path": "..\\..\\..\\Roslyn.sln",
+    "projects": [
+      "src\\Compilers\\CSharp\\CSharpAnalyzerDriver\\CSharpAnalyzerDriver.shproj",
```

Comment:
Rather than having the full closure here, can we just add the few projects that actually need to be built and let MSBuild build the dependencies? Or just have a traversal project for those?